src/components/courses/HandInExamsPage.tsx

```
@@ -0,0 +1,72 @@
+    }, "File format must be ZIP or RAR"),
+});
+
+const HandInExamsPage = () => {
+  const fields = [{ name: "exam", type: "file", accept: "" }];
+  const location = useLocation();
+  const state = location.state;
+  console.log(state);
+
+  const handleAddCourse = (formData: FormData) => {
+    let user: any;
+    let userJson = localStorage.getItem("user");
```

Biome

[error] This let declares a variable that is only assigned once.

'userJson' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)